helm/nuts-knooppunt/values.yaml

```
@@ -0,0 +1,166 @@
+# Default values for nuts-knooppunt.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# This will set the replicaset count more information can be found here: https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/
+replicaCount: 1
+
+# This sets the container image more information can be found here: https://kubernetes.io/docs/concepts/containers/images/
+image:
+  repository: ghcr.io/nuts/nuts-knooppunt/nuts-knooppunt
```

This will fail right?

```suggestion
  repository: ghcr.io/nuts-foundation/nuts-knooppunt
```